packages/service-library/src/servicelib/rabbitmq.py

```diff
@@ -226,7 +301,10 @@ async def rpc_request(
             raise e
 
     async def rpc_register_handler(
-        self, namespace: RPCNamespace, method_name: RPCMethodName, handler: Awaitable
+        self,
+        namespace: RPCNamespace,
+        method_name: RPCMethodName,
+        handler: Callable[..., Any],
```

Comment:
Question: why is this a callable and not an awaitable?

Reply:
because what RPC expects is a Callable, not an Awaitable.